spec/acceptance/suites/ce/20_multi_node_spec.rb

```diff
@@ -31,6 +31,8 @@ class { 'simp_docker':
     hosts.each do |host|
       it 'should apply with no errors' do
         apply_manifest_on(host, manifest, catch_failures: true, run_in_parallel: true)
```

Comment:
Is this expected behavior...to take 2 runs to be idempotent?

Reply:
Yes, once to install and start docker, one to set the sysctl values which are only available after the docker daemon starts